### ISSUE TITLE
Chopsticks integration

### ISSUE THREAD

**Issue by xlc** [polkadot-js/apps#9903]
It is now possible to run Chopsticks in browser ([example](https://acalanetwork.github.io/chopsticks/).
This means it is possible to have the ability to turn a mainnet to a devnet and then user can simulate transactions etc.

We can do the integration if desired.

**Comment by jacogr** [comment]
I know it is starting to gain traction and certainly fills a big need. With that in mind, PRs would be really welcome.

**Comment by qiweiii** [comment]
@xlc @jacogr I am thinking maybe we can add a page under 'Developer' called 'Dry Run', in there, user can set block and run extrinsics, any suggestions?

**Comment by xlc** [comment]
I can see a few integration points. We can add a Dry run button to the extrinsic page, next to Submit Unsigned. This will allow people to dry run the transaction and see the outcome.

Another way is on the network selection page, next to the Switch button, and a "Fork locally" button, and it will take the current endpoint as upstream and run a Chopsticks instance, expose an API and use that instead. Then change the color schema to something else and on the top nav bar add a text "Local testnet powered by Chopsticks" to indicate to users that they are not connecting a real network.

**Comment by xlc** [comment]
@jacogr Please take a review

**Comment by kianenigma** [comment]
What would be very useful is to also provide an easier way to fake any account once you click into that button. I am looking into the code to see if the CS instance is executed with `mock-signature-host` or not, but even if yes, atm I need to hack my way around. Ideally, the UI would facilitate this and allow you to:


1. Add any account you wish merely by the public key
2. (fake) Sign with all of your accounts.  

Without too much manual work needed.

**Comment by TarikGul** [comment]
Yup this can be closed (thought the PR was linked) @kianenigma Thanks :)
